### ISSUE TITLE
docs(readme): label the subagent quick benchmark simulation and preserve the live proof gate

### ISSUE THREAD

**Issue by anthony-chaudhary** [anthony-chaudhary/fak#12674]
## Problem

The Try fak section presents `fak bench subagent --concurrency=4` as a host measurement of four co-batched subagents, 18,000+ tokens/sec, greater than 95% cache hits, and bit-exact parity. The command defaults to `--simulated=true`: it exercises the context-MMU fork mechanism, but models GPU timings and cache traffic, synthesizes logits, and launches neither OpenCode workers nor a physical native model. Nearby prose also promises four fixed OpenCode roles and native co-batched decode without a committed end-to-end witness.

## Intended outcome

Keep the useful runnable demo while labeling its provenance precisely. Explain what the default simulation proves, point users toward the live `fak up` plus `fak opencode` path, and retain a clear promotion gate for measured OpenCode and native fanout claims.

## User benefit

Evaluators can reproduce the command and understand exactly what it demonstrates without mistaking architecture simulation for observed model throughput. The page still leads toward the stronger live product proof once its receipt exists.

## Classification

- Portfolio tier: 1, all-in-one serving plus harness plus memory.
- Centrality: Core documentation correctness.
- P1 context: preserve the Try fak path while separating simulation, mechanism, and live execution.
- P2 net value: prevent false evaluation conclusions from modeled rates.
- P3 adaptation: align README with the existing provenance field and physical runner gate.
- P4 operations: keep the command cheap and offline; require immutable receipts before restoring physical performance language.

## Relations

Coordinates with: #12551
Coordinates with: #12097
Coordinates with: #12307
Promotion requires: #12097

## Scope

`README.md` only. A later evidence note may be added only when the actual OpenCode/native run is captured.

## Acceptance

- The default benchmark is visibly labeled `[SIMULATED]` and its output provenance is explained.
- The text makes no observed tokens/sec, cache-hit, bit-exact, fixed-role, zero-cold-start, or native co-batching claim without a bound receipt.
- The live OpenCode/native run remains the required next proof rather than being treated as solved by the wording change.
- README freshness and appeal checks pass.

## Witness

`python3 tools/readme_freshness_audit.py`

`python3 tools/doc_appeal_scorecard.py --target README.md`
